<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(0,0,0);
}

QMenuBar{
background-color: rgb(0,0,0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>681</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color: rgb(0,0,0);
    border: 1px solid black;
    color: white; 
}</string>
    </property>
    <property name="title">
     <string>Items</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>621</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Search items</string>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox QAbstractItemView {
    background-color: rgb(30, 30, 40);
    color: #FFF;
    selection-background-color: rgb(53, 53, 71);
    selection-color: #FFF;
}


QComboBox{
	background-color: rgb(30, 30, 40);
	color: rgb(255,255,255);
	border: 2px solid rgb(37, 39, 48);
}

QComboBox:hover{
	border: 2px solid rgb(53, 53, 71);
	
	background-color: rgb(40, 40, 54);
}</string>
      </property>
      <item>
       <property name="text">
        <string>ItemID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Name</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="itemSearchLine">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>621</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    background-color: rgb(0,0,0);
    border: 1px solid black;
    color: white; 
}
</string>
     </property>
     <property name="title">
      <string>AddItems</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>ItemID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>Qty Available:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>Price:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="itemqtyAvailable">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="itemAddID">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="itemAddName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="itemAddPrice">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="InsertButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #FFF;
}</string>
      </property>
      <property name="text">
       <string>Colour:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="itemAddColour">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);

}

QLineEdit:hover{
	border: 2px solid rgb(53, 53, 71)
}

QLineEdit: focus{
	border: 2px solid rgb(85, 170, 255);
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>621</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    background-color: rgb(0,0,0);
    border: 1px solid black;
    color: white; 
}</string>
     </property>
     <property name="title">
      <string>ItemList</string>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>511</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget{
	background-color: rgb(0,0,0);
	gridline-color: rgb(255, 255, 255);
	color: rgb(255,255,255);
	border-radius: 10px;
	border: 2px solid rgb(255,255,255);
	selection-background-color: rgb(46, 46, 62);
}</string>
      </property>
      <column>
       <property name="text">
        <string>ItemID</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Colour</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Qty Available</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="UpdateButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeleteButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reportsButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>230</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
      </property>
      <property name="text">
       <string>Reports</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="BackButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border: 2px solid rgb(37, 39, 48);
	border-radius: 100px;
	color: #FFF;
	background-color: rgb(30, 30, 40);
	
}

QPushButton:hover{
	border: 2px solid rgb(53, 53, 71)
}</string>
     </property>
     <property name="text">
      <string>&lt;--</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
